Heimdall/Features/AccountOverview/AccountOverviewViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  AccountOverviewViewController.swift
+//  Heimdall
+//
+//  Created by Luis Reisewitz on 08.09.17.
+//  Copyright © 2017 Gnosis. All rights reserved.
+//
+
+import Bond
+import PureLayout
+import ReactiveKit
+import UIKit
+
+class AccountOverviewViewModel {
+    let accountLabelText: Property<String>
+    let balanceLabelText: Property<String>
+
+    let title = Property("Your Account")
+
+    let disposeBag = DisposeBag()
+
+    init(credentials: Credentials, rpc: EtherRPC) {
+        accountLabelText = Property("""
+            This is your account.
```

Comment:
should that be localized?

Reply:
Yes. Like every other string. Will do that asap (but not in this PR).